week-2/mandatory/3-playing-computer.js

```diff
@@ -1,42 +1,42 @@
 /*
   You have to predict the output of this program WITHOUT EXECUTING IT.
-
   In order to do this, try writing down the value that all variables take
   during each step of the program execution.
-
   Answer the following questions:
-
   1. This program throws an error. Why? (If you can't find it, try executing it).
+  - tha value b is not defind
   2. Remove the line that throws the error.
+  // console.log(b);
   3. What is printed to the console?
   4. How many times is "f1" called?
   5. How many times is "f2" called?
```

Comment:
Seems like you've missed out a couple of these. It might be worth to try and go through them still!